src/d3a/models/strategy/storage.py

```diff
@@ -15,208 +15,202 @@
-        break_even = {k: BreakEven(*v) for k, v in break_even.items()}
-        assert all(type(v) is BreakEven for k, v in break_even.items())
-        if any(be.sell < be.buy for _, be in break_even.items()):
-            raise ValueError("Break even point for sell energy is lower than buy energy.")
-        if any(break_even_point.buy < 0 or break_even_point.sell < 0
-               for _, break_even_point in break_even.items()):
-            raise ValueError("Break even point should be positive energy rate values.")
-        if any(break_even_point.sell > initial_selling_rate
-               for break_even_point in break_even.values()):
-            raise ValueError(f"Break even point(s) (selling) should be less than or equal "
-                             f"to initial_selling_rate ({initial_selling_rate})")
-        if ConstSettings.IAASettings.MARKET_TYPE != 1:
-            if any(break_even_point.buy < initial_buying_rate
-                   for break_even_point in break_even.values()):
-                raise ValueError(f"Break even point(s) (buying) should be greater than or equal "
-                                 f"to initial_buying_rate ({initial_buying_rate})")
-
-        return break_even
-
-    @staticmethod
-    def _validate_constructor_arguments(risk, initial_capacity_kWh, initial_soc,
-                                        battery_capacity_kWh, min_allowed_soc,
-                                        initial_selling_rate):
-        if battery_capacity_kWh and battery_capacity_kWh < 0:
+    def _validate_constructor_arguments(initial_soc=None, min_allowed_soc=None,
+                                        battery_capacity_kWh=None, max_abs_battery_power_kW=None,
+                                        initial_selling_rate=None, final_selling_rate=None,
+                                        initial_buying_rate=None, final_buying_rate=None,
+                                        energy_rate_change_per_update=None):
+        if battery_capacity_kWh is not None and battery_capacity_kWh < 0:
             raise ValueError("Battery capacity should be a positive integer")
-        if initial_soc and not min_allowed_soc <= initial_soc <= 100:
-            raise ValueError("Initial charge is a percentage value, should be between "
-                             "MIN_ALLOWED_SOC and 100.")
-        if risk and not 0 <= risk <= 100:
-            raise ValueError("Risk is a percentage value, should be between 0 and 100.")
-        if initial_capacity_kWh and battery_capacity_kWh and min_allowed_soc and \
-           not min_allowed_soc * battery_capacity_kWh <= \
-           initial_capacity_kWh <= battery_capacity_kWh:
-            raise ValueError(f"Initial capacity should be between min_allowed_capacity and "
-                             "battery_capacity_kWh parameter.")
-        if initial_selling_rate < 0:
+        if max_abs_battery_power_kW is not None and max_abs_battery_power_kW < 0:
```

Comment:
Probably not part of this story, but these validation methods would be nice to be on the d3a-interface, so that d3a-web can take advantage of these too. Not necessary for now, if you agree (also @hannesdiedrich ) we can open a new story for merging the validation functions for these parameters in d3a-interface.